hack/jenkins/test-dockerized.sh

```diff
@@ -40,7 +40,6 @@ export KUBE_COVER="n"
 export ARTIFACTS=${ARTIFACTS:-"${WORKSPACE}/artifacts"}
 # Save the verbose stdout as well.
 export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
-export KUBE_INTEGRATION_TEST_MAX_CONCURRENCY=4
 export LOG_LEVEL=4
 
 cd "${GOPATH}/src/k8s.io/kubernetes"
```

Comment:
this is super brittle and really we should basically eliminate this script such that `make test-integration` just does the right thing locally and in CI, but we'll save that for future PRs

Reply:
NOTE: we don't call this one on the master branch anymore, only in <=release-1.32

https://cs.k8s.io/?q=test-dockerized.sh&i=nope&literal=nope&files=&excludeFiles=&repos=

https://github.com/kubernetes/test-infra/pull/34394

we do however use test-integration-dockerized.sh

this script can probably be deleted, I'd like to make that a seperate, revertible PR though as we only recently made the CI change

Reply:
I've started a larger PR for cleanup, but forked off to prioritize merging automaxprocs

Reply:
that will be https://github.com/kubernetes/kubernetes/pull/130760